frontend/packages/dev-console/src/utils/pipeline-utils.ts

```diff
@@ -56,6 +56,18 @@ export const conditions = {
     task.runAfter && task.runAfter.length > 0,
 };
 
+export enum ListFilterIds {
```

Comment:
I believe *enum*s are supposed to be singular. 
```suggestion
export enum ListFilterId {
```